docs/deployment/play-store.mdx

```diff
@@ -69,15 +69,15 @@ If you haven't made the switch to Android app bundles yet, you will need to opt
 First, create a new app in the Google Play Console. In order to enable app signing, you'll need to navigate to the new release screen, on one of the Production, Open testing, closed testing, or internal testing pages.
 Select the Create new release button as seen below (it doesn't matter which release type, since you don't have to actually go through with creating a new release right now):
 
-![Select create new release in Google Play Console's Production tab](https://blog.ionicframework.com/wp-content/uploads/2021/12/newapps-release-1024x561.png)
+![Screenshot of Google Play Console's Production tab with arrow pointing to the 'Create new release' button.](https://blog.ionicframework.com/wp-content/uploads/2021/12/newapps-release-1024x561.png "Google Play Console New Release Creation")
```

Comment:
As mentioned above, it's best to avoid terms like "screenshot of" for images. I've already made requests for alterations to some images. I'll refrain from requesting changes to the remaining ones to prevent overwhelming you with repetitive requests. Regardless, I strongly recommend avoiding such terms in all files.